pkg/getty/Dockerfile

```diff
@@ -9,6 +9,9 @@ RUN apk add --no-cache --initdb -p /out \
     util-linux \
     && true
 RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+RUN cp /out/usr/bin/setsid /out/usr/bin/setsidu
```

Comment:
can you just add a comment about why this is.

We should be able to remove it once `init` no longer uses `busybox` which is not that far off.